_data/testimonials.yml

```diff
@@ -12,19 +12,19 @@
   
 - name: Carlos Pedroza
   username: carlospedroza27
-  text: Makeroid is merging the ability to create awesome android applications without writing a single line of code and the ability to make a professional app in a stunning way.
+  text: Makeroid is merging the ability to create awesome Android apps without writing a single line of code and the ability to make an app in a stunning way.
 
-- name: Jorge
+- name: Jorge VB
   username: J.V.B
-  text: The first time I tried Makeroid, I was really impressed with the number of components that Makeroid offered me. It’s much more flexible than others, and there’s always a community waiting to hear your problems, ideas, or suggestions. It’s also very easy to learn, even for the most beginner.
+  text: The first time I tried Makeroid, I was impressed with the number of components that it got. It’s also very easy to learn, even for the most beginners.
```

Comment:
A couple of parts here don't make much sense. Can you change `it got` to `it has got`? Can you remove `the` from `even for the most beginners`?

Reply:
Fixed @conorshipp.